frontend/src/widgets/WidgetHelper.js

```diff
@@ -0,0 +1,18 @@
+/* WidgetHelper.js */
+/* Helper Functions for Widgets */
+function FormatNumber(numberToFormat, decimalPlaces) {
+  let decimalPlacesToUse = decimalPlaces;
+  if (!numberToFormat || Number.isNaN(numberToFormat)) {
+    return 0;
+  }
+  let number = parseFloat(numberToFormat);
+  if (!decimalPlacesToUse) {
+    decimalPlacesToUse = 0;
```

Comment:
`decimalPlacesToUse` isn't used, do you mean to use it on line 14? If so you can use a default arg:
```
function FormatNumber(numberToFormat, decimalPlaces = 0) {
```

Reply:
This has been changed.